### PR TITLE
Fixing the spacing and tooltip for breakdown card in storage dashboards

### PR DESCRIPTION
**Pull request by afreen23**
**BEFORE**
Screenshot from 2019-11-11 15-13-45
**AFTER**
Screenshot from 2019-11-11 15-01-36
Screenshot from 2019-11-11 15-06-15

### CODE REVIEW

**Comment by spadgett** [comment]
Last failure was https://bugzilla.redhat.com/show_bug.cgi?id=1765280

/retest

File: frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/breakdown-card/breakdown-card.scss [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 .capacity-breakdown-card__capacity-body {
   font-size: var(--pf-global--FontSize--sm);
+  margin-bottom: 0rem;
```
Comment (gnehapk) [inline frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/breakdown-card/breakdown-card.scss]: No need to add `rem` unit here as its 0

File: frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/breakdown-card/breakdown-chart.tsx [hunk 2]
```diff
@@ -41,49 +38,46 @@ export const BreakdownChart: React.FC<BreakdownChartProps> = ({ data, legends, m
...
       style={{ data: { stroke: 'white', strokeWidth: 0.7, ...isAvailableBar(d.name) } }}
       cornerRadius={getBarRadius(index, data.length)}
-      barWidth={20}
+      barWidth={18}
       padding={0}
       data={[d]}
-      labelComponent={
-        <ChartTooltip constrainToVisibleArea style={{ fontSize: 8 }} pointerOrientation="left" />
-      }
+      labels={({ datum }) => `${datum.name}: ${datum.y}`}
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/breakdown-card/breakdown-chart.tsx]: Will remove
```suggestion
```

**Comment by andybraren** [comment]
This is looking great! 👍 One small thing; the top-left and bottom-left corners of the chart don't seem to have the same corner radius as the top-right and bottom-right corners. All four corners should be slightly rounded.

**Comment by afreen23** [comment]
> This is looking great!  One small thing; the top-left and bottom-left corners of the chart don't seem to have the same corner radius as the top-right and bottom-right corners. All four corners should be slightly rounded.

@andybraren here the capacity of first project is lesser in comparison to the rest of "Available".
The first one ( openshift ) is rounded but due to lesser capacity its not significantly visible.

**Comment by andybraren** [comment]
Ah thanks @afreen23 that makes sense, and sorry I didn't realize this had already merged before commenting, I think this is totally okay then (it seems relatively rare)! 🙂

**Pull request merged**